### ISSUE TITLE
Invalid path for receipts

### ISSUE THREAD

**Issue by paukert** [Quick-Event/quickbox#946]
- although Reports path is set correctly in settings and e.g. start lists can be printed, the receipts path is not set correctly (it is not possible to preview the receipt or select receipt which should be used)
- problem is probably connected to changing Reports directory - receipts path is not refreshed correctly (when QuickEvent is relaunched, the correct path for receipts is set)
- tested with AppImage build, version: https://github.com/Quick-Event/quickbox/actions/runs/5980024579
- when the Receipts tab is selected in Settings, the following line is printed to the log
```
2023-08-27T13:06:17[receiptssettingspage.cpp:45]|I| current receipt path:
```
![obrazek](https://github.com/Quick-Event/quickbox/assets/15065907/3f0cd24d-3148-487c-9a65-e239b80194d4)

**Comment by paukert** [comment]
Isn't current "dev" branch for QE3/QT6 `qe3`? Fix for this issue was pushed to `qt6` branch.

**Comment by fvacek** [comment]
> Isn't current "dev" branch for QE3/QT6 `qe3`? Fix for this issue [9de2bed](https://github.com/Quick-Event/quickbox/commit/9de2bed6537e3a62cae03efede9750b8824236a3) and commits [9a45401](https://github.com/Quick-Event/quickbox/commit/9a45401deff66b66c96558083665cd915d0140c5), [de2d139](https://github.com/Quick-Event/quickbox/commit/de2d1394d2449836a6c1cb052f31e3aa0893f1a3) were pushed to the `qt6` branch.

sorry, my fault, thank you, current dev branch is `qe3`

**Comment by fvacek** [comment]
> Isn't current "dev" branch for QE3/QT6 `qe3`? Fix for this issue [9de2bed](https://github.com/Quick-Event/quickbox/commit/9de2bed6537e3a62cae03efede9750b8824236a3) and commits [9a45401](https://github.com/Quick-Event/quickbox/commit/9a45401deff66b66c96558083665cd915d0140c5), [de2d139](https://github.com/Quick-Event/quickbox/commit/de2d1394d2449836a6c1cb052f31e3aa0893f1a3) were pushed to the `qt6` branch.

cherry-picked and pushed to `qe3`

**Comment by paukert** [comment]
Fixed in 3.0.5
